srcjs/methods/selectize.ts

```diff
@@ -0,0 +1,61 @@
+import { $ } from "../globals";
+
+function click(id: string) {
+  $(`#${id}`)
+    .siblings()
+    .filter(".selectize-control")
+    .find(".selectize-input")
+    .click();
+}
+
+function options(id: string) {
+  return $(`#${id}`)
+    .siblings()
+    .filter(".selectize-control")
+    .find(".selectize-dropdown-content")
+    .children();
+}
+
+function clickOption(id: string, idx: number) {
+  const opt = options(id).get(idx);
+
+  if ($(opt).hasClass("optgroup")) {
+    $(opt)
+      .find(".option")
+      .click();
+  } else {
+    opt.click();
+  }
+}
+
+function currentChoice(id: string) {
+  return $(`#${id}`)
+    .siblings()
+    .filter(".selectize-control")
+    .find(".selectize-input")
+    .text();
+}
+
+function values(id: string) {
```

Comment:
Should have a comment saying server-side selectize provides the first 1000 choices.